### PR TITLE
Add lots of styling and copy tweaks actueel page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Lots of design tweaks for the actueel page.

Including:
- Added on municipality actueel a risk level indicator 
- Design tweaks on the risk level indicator
- Added "gemeente" and "veiligheidsregio" in the quicklinks also in the title of the page
- Reduced safetyregion chloropleth  width and added some padding
- Added on municipality actueel a safety region text with button
- The rest are small font and color tweaks

**Additional**

Added *silver* to the theme colors. Well, why the name silver? 
https://chir.ag/projects/name-that-color/#C4C4C4

And since light gray and gray already exists I don't wanna add more confusion.

### CODE REVIEW

File: packages/app/src/pages/index.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ const Home: FCWithLayout<typeof getStaticProps> = (props) => {
                 <Box
                   borderTopWidth="1px"
                   borderTopStyle="solid"
-                  borderTopColor="gray"
+                  borderTopColor="#C4C4C4"
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/index.tsx:220]: Let's add this to the theme

File: packages/app/src/components-styled/risk-level-indicator.tsx [hunk 2]
```diff
@@ -59,12 +59,15 @@ export function RiskLevelIndicator(props: RiskLevelIndicatorProps) {
...
+            {escalationLevel}
+          </Box>
+          {escalationTypes[escalationLevel].titel}
         </Box>
       </Box>
 
-      <Text mb="0">{description}</Text>
-      <Text mt="0">
+      <Text>
+        {`${description} `}
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/components-styled/risk-level-indicator.tsx:70]:
```suggestion
        {`${description} ${escalationLevel}: `}
```

or 

```
        {description}
        {' '}
        {escalationLevel}
        {': '}
```

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: packages/app/src/components-styled/risk-level-indicator.tsx (hunk 2 above)
Comment (0x80) [reply packages/app/src/components-styled/risk-level-indicator.tsx:70]: I'm in favor of the first suggestion

**Review by 0x80 — APPROVED** [review]

File: packages/app/src/components-styled/article-teaser.tsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ export function ArticleTeaser(props: ArticleLinkProps) {
     <Box
       border="solid"
       borderWidth={1}
-      borderColor="border"
+      borderColor="#dcdcdc"
```
Comment (VWSCoronaDashboard1) [inline packages/app/src/components-styled/article-teaser.tsx:31]: perhaps add this color to the theme as well?

File: packages/app/src/components-styled/line-chart/components/chart-axes/index.tsx [hunk 4]
```diff
@@ -25,7 +26,7 @@ export const defaultPadding: ChartPadding = {
 };
 
 const defaultColors = {
-  axis: '#C4C4C4',
+  axis: colors.silver,
   axisLabels: '#666666',
   benchmark: '#4f5458',
```
Comment (VWSCoronaDashboard1) [inline packages/app/src/components-styled/line-chart/components/chart-axes/index.tsx:31]: Same for these. There's a 'data' key in the colors lookup. I'm guessing these can be added there.

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Pull request merged**